go.mod

```diff
@@ -14,6 +14,7 @@ replace (
 
 	github.com/rancher/rancher/pkg/apis => ./pkg/apis
 	github.com/rancher/rancher/pkg/client => ./pkg/client
+	github.com/rancher/shepherd => github.com/igomez06/shepherd v0.0.0-20240819163615-695ebe4c12b0
```

Comment:
this still needs to be updated now that shepherd has merged, waiting on this for approval.